regulation-worker/cmd/main.go

```diff
@@ -33,14 +34,13 @@ func main() {
 func Run(ctx context.Context) {
 	svc := service.JobSvc{
 		API: &client.JobAPI{
-			WorkspaceID: getEnv("workspaceID", "1001"),
-			URLPrefix:   getEnv("urlPrefix", "http://localhost:35359"),
+			WorkspaceID: config.GetEnv("workspaceID", "1001"),
+			URLPrefix:   config.GetEnv("urlPrefix", "http://localhost:35359"),
```

Comment:
Maybe we should give `https://api.rudderlabs.com:35359` as default value

Reply:
makes sense